### PR TITLE
[CPU] OneDNN 2.6 migration

### PR DESCRIPTION
**Pull request by luo-cheng2021**
### Details:
 - *OneDNN 2.6 migration*
 - *...*

### Tickets:
 - *81372*

### CODE REVIEW

File: src/plugins/intel_cpu/src/nodes/def_conv.cpp [hunk 1]
```diff
@@ -118,7 +118,7 @@ struct jit_uni_def_conv_kernel_f32 : public jit_uni_def_conv_kernel, public jit_
     Xbyak::Label l_table;
 
     inline void checkZeroWei(const Xbyak::Xmm &x1, Label &nullifyLabel) {
-        uni_vtestps(x1, x1);
+        ptest(x1, x1);
```
Comment (luo-cheng2021) [inline src/plugins/intel_cpu/src/nodes/def_conv.cpp:121]: Need confirm. @zhangYiIntel

Comment (zhangYiIntel) [reply src/plugins/intel_cpu/src/nodes/def_conv.cpp:121]: I check the uni_vtestps implementation, the `uni_vtestps`  only wraps ptest

```
    void uni_vtestps(const Xbyak::Xmm &x1, const Xbyak::Operand &op) {
        ptest(x1, op);
    }
```

However, the oneDNN2.6 has change the implementation which generates code we don't want.
```
    void uni_vtestps(const Xbyak::Xmm &x1, const Xbyak::Operand &op) {
        if (is_valid_isa(avx))
            vptest(x1, op);
        else
            ptest(x1, op);
    }
```

File: src/plugins/intel_cpu/src/nodes/memory.cpp [hunk 2]
```diff
@@ -136,12 +137,17 @@ inline
 static void simple_copy(const Memory& dst, const Memory& src) {
     auto srcPtr = static_cast<uint8_t*>(src.GetPtr());
     auto dstPtr = static_cast<uint8_t*>(dst.GetPtr());
-    auto srcSizeInByte = src.GetSize();
-    auto dstSizeInByte = dst.GetSize();
+    if (src.GetDataType() == dst.GetDataType()) {
```
Comment (luo-cheng2021) [inline src/plugins/intel_cpu/src/nodes/memory.cpp:140]: Use cpu_convert instead of cpu_memcpy.

File: src/plugins/intel_cpu/src/plugin.cpp [hunk 3]
```diff
@@ -407,7 +407,7 @@ static void TransformationUpToCPUSpecificOpSet(std::shared_ptr<ngraph::Function>
     pass_config->disable<ngraph::pass::ConvertGather8ToGather7>();
     pass_config->disable<ngraph::pass::ConvertMinimum>();
     pass_config->disable<ngraph::pass::ConvertBroadcastToTiles>();
-    pass_config->disable<ngraph::pass::ConvertReduceMeanToPooling>();
+//    pass_config->disable<ngraph::pass::ConvertReduceMeanToPooling>();
```
Comment (luo-cheng2021) [inline src/plugins/intel_cpu/src/plugin.cpp:410]: Add todo comments

File: src/plugins/intel_cpu/src/plugin.cpp [hunk 4]
```diff
@@ -497,7 +497,7 @@ static void TransformationUpToCPUSpecificOpSet(std::shared_ptr<ngraph::Function>
                 WeightableLayerTransformation::isAsymmetricOnWeights(node, defaultPrecisions);
         });
         lptManager.get_pass_config()->set_callback<ngraph::pass::low_precision::MultiplyToGroupConvolutionTransformation>([](const_node_ptr& node) -> bool {
-            return MultiplyToGroupConvolutionTransformation::isDynamicOrScalar(node);
+            return true;//MultiplyToGroupConvolutionTransformation::isDynamicOrScalar(node);
```
Comment (luo-cheng2021) [inline src/plugins/intel_cpu/src/plugin.cpp:500]: Add todo comments.

File: src/plugins/intel_cpu/src/utils/debug_capabilities.cpp [hunk 5]
```diff
@@ -0,0 +1,341 @@
...
+
+    // last line(s): fused layers
+    os << " " << node.getOriginalLayers();
+
+    if (node.PerfCounter().count()) {
+        os << " latency:" << node.PerfCounter().avg() << "(us) x" << node.PerfCounter().count();
+    }
+
+    // primArgs
+    /*
```
Comment (luo-cheng2021) [inline src/plugins/intel_cpu/src/utils/debug_capabilities.cpp:321]: Remove

File: src/plugins/intel_cpu/src/nodes/memory.cpp (hunk 2 above)
Comment (luo-cheng2021) [reply src/plugins/intel_cpu/src/nodes/memory.cpp:140]: Will fix in next pr.

File: src/plugins/intel_cpu/src/utils/debug_capabilities.cpp (hunk 5 above)
Comment (luo-cheng2021) [reply src/plugins/intel_cpu/src/utils/debug_capabilities.cpp:321]: Will fix in next pr.

File: src/plugins/intel_cpu/src/plugin.cpp (hunk 3 above)
Comment (luo-cheng2021) [reply src/plugins/intel_cpu/src/plugin.cpp:410]: Will fix in next pr.

File: src/plugins/intel_cpu/src/plugin.cpp (hunk 4 above)
Comment (luo-cheng2021) [reply src/plugins/intel_cpu/src/plugin.cpp:500]: Will fix in next pr.

**Review by usstq — APPROVED** [review]

**Pull request merged**